src/native/eventpipe/ep-session.c

```diff
@@ -332,7 +332,6 @@ ep_session_enable_rundown (EventPipeSession *session)
 	EventPipeProviderConfiguration rundown_providers [2];
```

Comment:
Should be 1 rundown_providers.

Reply:
Thanks, fixed this to get rid of the loop since there is just one provider now